db/schema/4-person.sql

```diff
@@ -40,4 +40,12 @@ CREATE TABLE IF NOT EXISTS person_avinya_type_transition_history (
     FOREIGN KEY (person_id) REFERENCES person(id),
     FOREIGN KEY (previous_avinya_type_id) REFERENCES avinya_type(id),
     FOREIGN KEY (new_avinya_type_id) REFERENCES avinya_type(id)
+);
+
+CREATE TABLE IF NOT EXISTS parent_child_student(
+    child_student_id INT NOT NULL,
```

Comment:
Nit: Remove the `_student_` from both the `child_student_id` and `parent_student_id`. The parent/child concept is independent of whether or not the child is a student